hastory-cli/src/Hastory/Cli/OptParse.hs

```diff
@@ -153,4 +169,20 @@ parseFlags =
              , metavar "DIR"
              , value Nothing
              , help "the cache directory for hastory"
+             ]) <*>
+    option
+        (Just . T.pack <$> str)
+        (mconcat
+             [ long "storage-server-url"
+             , metavar "URL"
+             , value Nothing
+             , help "URL of the central storage server"
+             ]) <*>
+    option
+        (Just . T.pack <$> str)
+        (mconcat
+             [ long "storage-server-token"
+             , metavar "TOKEN"
```

Comment:
Yes, I'm really not happy with this kind of "authentication". I just wanted to prevent someone from being able to do arbitrary POST requests to append server to send bogus data.